src_c/color.c

```diff
@@ -1423,6 +1441,46 @@ _color_set_i1i2i3(pgColorObject *color, PyObject *value, void *closure)
     return 0;
 }
 
+static PyObject *
+_color_get_rgb(pgColorObject *color, void *closure)
+{
+    return Py_BuildValue("(iii)", color->data[0], color->data[1],
```

Comment:
Just a question, with my recent PR #3401 i discovered that creating a tuple, setting its arguments and returning it is about 40% faster than using `Py_BuildValue`, so can we implement this here already insted of opening another PR later? Do we need/want to do this?

Reply:
@MyreMylar Thank you, so I'm waiting for a merge of that.